pkgs/top-level/python-packages.nix

```diff
@@ -715,6 +715,8 @@ in {
 
   mailman = disabledIf (!isPy3k) (callPackage ../servers/mail/mailman/core.nix { });
 
+  mailman-web = disabledIf (!isPy3k) (callPackage ../servers/mail/mailman/web.nix { });
```

Comment:
This is not the right place. `python-packages.nix` should only contain libraries. This is configuration for the mailman module and should be situated there (it is) and be called from there (which it is not currently).

Reply:
What do you mean by "there" exactly? Where should that definition be in your opinion?

Reply:
in nixos/modules/services/mail/mailman.nix

Reply:
I don't know ... I am not convinced that defining that module exclusively for NixOS is an improvement.

Reply:
I turned mailman-web into a proper python package in 86f8895abbf.